asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -100,8 +102,11 @@ def __init__(
         self.output_paths = []
         # make sure all paths exist, otherwise skip
         for pose, path in zip(poses, output_paths):
-            if pose and Path(pose).exists():
-                mol = load_openeye_sdf(str(pose))
+            if pose:
+                if isinstance(pose, oechem.OEMolBase):
+                    mol = pose.CreateCopy()
+                else:
+                    mol = load_openeye_sdf(str(pose))
```

Comment:
`load_openeye_sdf` takes `Path` as input now
```suggestion
                    mol = load_openeye_sdf(pose)
```

Reply:
See above RE Oemol / Ligand.